packages/dropdowns.next/src/views/combobox/StyledFloatingListbox.ts

```diff
@@ -37,7 +36,6 @@ export const StyledFloatingListbox = styled.div.attrs({
     menuStyles(props.position, {
       theme: props.theme,
       hidden: props.isHidden,
-      childSelector: `> ${StyledListbox}`,
```

Comment:
I'm curious, was this removal necessary to get animation working?

Reply:
In this case, yes. Since `FloatingMenu` extends `FloatingListbox`, and both these containers always have one child, removing this line allows the [direct descendent fallback](https://github.com/zendeskgarden/react-components/blob/main/packages/theming/src/utils/menuStyles.ts#L47) to work for both components.